packages/notebook/src/widget.ts

```diff
@@ -1294,6 +1294,7 @@ export namespace StaticNotebook {
      */
     windowingMode: 'defer' | 'full' | 'none' | 'contentVisibility';
     accessKernelHistory?: boolean;
+    showMinimap: boolean;
```

Comment:
Too lessen migration cost can we make it optional? Also it needs a documentation above (as does `accessKernelHistory` but it's a separate problem!)


```suggestion
    showMinimap?: boolean;
```